app/views/layouts/_footer.html.erb

```diff
@@ -6,11 +6,12 @@
 		</div>
 	</div><!-- end row -->
 </div><!-- end container -->
-	  
+
 <div id="app_footer" class="container text-center margin-bottom-sm">
   <a href="http://<%= FUND_DOMAIN %>/"><%= FUND_FULL %></a> -- <a href="tel:<%= FUND_PHONE %>"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> <%= FUND_PHONE %></a><br>
 	<a href="https://prochoice.org/">National Abortion Federation</a> -- <a href="tel:800-772-9100"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> 800-772-9100</a><br>
-	<%= link_to FUND_FAX_SERVICE, "http://#{FUND_FAX_SERVICE}", target: '_blank', rel: 'noopener nofollow' %>
+	<a href="http://<%= fax_service %>/"><%= fax_service %></a> -- <a href="tel:<%= fax_number %>"><span class="glyphicon glyphicon-earphone" aria-hidden="true"></span> <%= fax_number %></a><br>
```

Comment:
Rails has a pretty sick helper for cleaning up this kind of thing - https://api.rubyonrails.org/v5.2.1/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

So we can effectively replace this with `link_to nil, fax_service, target:'_blank', rel: 'noopener nofollow'` and it'll save us having to do http etc.

```suggestion
	<%= link_to nil, fax_service, target: '_blank', rel: 'noopener nofollow' %>
```